reference/sectionD_references.inc

```diff
@@ -0,0 +1,17 @@
+
+.. _sectionD:
+
+Extensible Markup Language (XML) 1.1 (Second Edition) http://www.w3.org/TR/2006/REC-xml11-20060816 (16 August 2006)
+
+Mathematical Markup Language (MathML) Version 2.0 (Second Edition) http://www.w3.org/TR/2003/REC-MathML2-20031021/ (21 October 2003)
```

Comment:
Prefer https://www.w3.org/TR/2003/REC-MathML2-20031021/.

Reply:
why is this not https://www.w3.org/TR/MathML2/ ? Surely if an updated MathML 2 spec was released we would be using it...

Reply:
and a reference list would generally have the date the reference was accessed for an online reference, not the date the resource was published.

Reply:
> why is this not https://www.w3.org/TR/MathML2/ ? Surely if an updated MathML 2 spec was released we would be using it...

I was merely suggesting replacing `http` with `https`, not thinking about the URL itself, but I agree that https://www.w3.org/TR/MathML2/ is better.

> and a reference list would generally have the date the reference was accessed for an online reference, not the date the resource was published.

Not quite sure what you mean by this. Are you saying that we should have something like 1 April 2020 instead?

Reply:
> Not quite sure what you mean by this. Are you saying that we should have something like 1 April 2020 instead?

Yep, you'd normally have (Accessed: 1 April 2020) or similar. But I don't think we need dates as discussed in the line 4 thread :)